src/common/components/Button/Button.tsx

```diff
@@ -1,35 +1,47 @@
 import React, { FC } from "react";
 
 import arrow from "@Images/arrow_right.svg";
+import plus from "@Images/plus.svg";
+import heart from "@Images/heart.svg";
 
 import "./Button.scss";
 
 interface ButtonProps {
   text: string;
-  onCLick: () => void;
+  onCLick?: () => void;
+  style?: "primary" | "secondary";
+  image?: "arrow" | "plus" | "heart";
+  imagePosition?: "right" | "left";
   arrowDirection?: "right" | "bottom";
   disabled?: boolean;
 }
 
 const Button: FC<ButtonProps> = ({
   text,
-  arrowDirection = "right",
   onCLick,
+  style = "primary",
+  image = "arrow",
+  imagePosition = "right",
+  arrowDirection = "right",
```

Comment:
enums?

Reply:
Fixed - add enums, replace strings with enums in components